lib/types.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module '@pkgjs/parseargs' {
+	export { parseArgs } from 'util'
+}
```

Comment:
I added these type definitions in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61997, but it’s pending code review.